### ISSUE TITLE
can you provide the loss log

### ISSUE THREAD

**Issue by valencebond** [OpenDriveLab/maskalign#3]
I use your code to pretrain video datasets, But I'm not sure what is the normal rate of loss decline

**Comment by HellwayXue** [comment]
Hi, due to the internship change, I have no access to the previous log. But reproducing Maskalign is very easy, so my suggestion is first pre-training on ImageNet and then checking the loss status. You can also discuss more details with me.
